edkrepo/common/ui_functions.py

```diff
@@ -14,6 +14,11 @@
 import git
 import colorama
 
+from colorama import Fore
+from colorama import Style 
+from colorama import init
+init(autoreset=True)

```

Comment:
I would recommend referring to this as colorama.init as in the init_color_console method so we avoid confusion about what init is being used.

I also think that we should move this to the call to colorama.init() in the init_color_console method and then call this method at the start of any command that uses these print methods